src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -120,6 +120,32 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             }
         });
 
+        _updateScrollBar = std::make_shared<ThrottledFunc<ScrollBarUpdate>>(
+            [weakThis = get_weak()](const auto& update) {
+                if (auto control{ weakThis.get() })
+                {
+                    control->Dispatcher()
+                        .RunAsync(CoreDispatcherPriority::Normal, [=]() {

```

Comment:
I am not clear on how reference and value captures work in this case. We got `const auto& update`above as a reference; will the `[=]` capture in the inner lambda promote (demote?) that to a plain value?

Reply:
Yes https://stackoverflow.com/questions/19676473/in-lambda-how-reference-is-being-captured-by-value